packages/react/async-boundary/src/AsyncBoundary.tsx

```diff
@@ -11,25 +11,12 @@ type Props = SuspenseProps &
     pendingFallback: ComponentProps<typeof Suspense>['fallback'];
   };
 
-interface ResetRef {
-  reset?(): void;
-}
```

Comment:
I delete unnecessary ResetRef. I think this interface need to be driven from ErrorBoundary.